examples/calculator/src/calculator.nit

```diff
@@ -0,0 +1,114 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Portable calculator UI
+module calculator is
+app_name "app.nit Calc."
+	app_version(0, 1, git_revision)
+	app_namespace "org.nitlanguage.calculator"
+
+	# Lock in portrait mode
+	android_manifest_activity """android:screenOrientation="portrait""""
+	android_api_target 15
+end
+
+import app::ui
+import app::data_store
+import android::aware
```

Comment:
Jean-Claude Van Damme? or are you suggesting that I move this import to the `app` or `app::ui` module?